roadmap.md

```diff
@@ -4,154 +4,95 @@ This document defines a high level roadmap for KubeEdge development.
 
 The [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) represent the most up-to-date plans.
 
-The roadmap below outlines KubeEdge’s 2023 feature plan.
+The roadmap below outlines KubeEdge’s 2024 feature plan.
 
-## 2023 H1 
+## SIG Node
 
-### SIG Node
+- Continuous follow up Kubernetes release
+- Support edge nodes running on mac OS and RTOS 
+- Support in-cluster config for edge pods to access Kube-APIServer
+- Enhancements to device plugin on edge nodes, such as support for multiple virtual GPUs
```

Comment:
AFAIS, `DevicePlugin` can supported on egecore already. what is `multiple virtual GPUs` by here, could you elaborate a bit about current problem?

Reply:
> AFAIS, `DevicePlugin` can supported on egecore already. what is `multiple virtual GPUs` by here, could you elaborate a bit about current problem?

EdgeCore do support DevicePlugin now. However, for some special device plugins, it seems that there are still some problems in use, so we need to do some testing and verification work. vGPUs are still in the research and verification phase.